src/main.py

```diff
@@ -4,16 +4,6 @@
 from operations import summation, subtraction
 
 
-# This is some code that doesn't work, and therefore
-# it is commented out. We should remove it at
-# some point:
-
-# def perform_operation(num1, num2, operation):
-#     if operation = "add":
-#         result = summation(num1, num2)
-
-#     return result
-
 def perform_operation(num1, num2, operation):
```

Comment:
Please add documentation for this function.

Reply:
Alright!